### PR TITLE
make readme awesome

### PR DESCRIPTION
**Pull request by thecodemonkey**
## Description
our readme file is unfortunately not yet quite awesome. 
we should create a new one. A bit cleaner, less content and only the most important.
Here are some inspirations:

https://github.com/matiassingers/awesome-readme

## Motivation and Context
Every **awesome** open source project needs an **awesome** readme file!

### CODE REVIEW

**Comment by thecodemonkey** [comment]
@NLueg @lionelpa @jages @sbischoff-ai 

The new readme is done: 

https://github.com/openvalidation/openvalidation/tree/make-readme-awesome

It would be cool if someone could proofread the days.

**Comment by thecodemonkey** [comment]
I have now deleted some line breaks so that it looks a bit more compact. But it is important to have some space between the chapters. It makes reading easier and looks better optically. 

Are there any other problems with grammar or content?

File: README.md [hunk 1]
```diff
@@ -1,140 +1,375 @@
...
+## introduction
+<br/>
 
-### Download & Installation
-Download the [openVALIDATION CLI](https://downloadarchive.blob.core.windows.net/openvalidation-generator/openvalidation.jar) (requires the Java SE 8 runtime environment).
+<a href="https://openvalidation.io" target="_blank" rel="noopener noreferrer"><img width="200" src="docs/ci/logo-v2.png" alt="a natural language compiler for validation rules"></a>
+
+<br/>
+
+openVALIDATION enables programming validation rules using natural language, such as German or English.
```
Comment (NLueg) [inline README.md]:
```suggestion
openVALIDATION enables programming validation rules using natural languages, such as German or English. 
```

File: README.md [hunk 2]
```diff
@@ -1,140 +1,375 @@
...
 
-### Download & Installation
-Download the [openVALIDATION CLI](https://downloadarchive.blob.core.windows.net/openvalidation-generator/openvalidation.jar) (requires the Java SE 8 runtime environment).
+<a href="https://openvalidation.io" target="_blank" rel="noopener noreferrer"><img width="200" src="docs/ci/logo-v2.png" alt="a natural language compiler for validation rules"></a>
+
+<br/>
+
+openVALIDATION enables programming validation rules using natural language, such as German or English. 
+
+The rules recorded in natural language are readable not only by humans but also by the machines and therefore no longer need to be programmed by a software developer. This task is now taken over by openVALIDATION compiler. With integrated code generators, corresponding source code in the desired programming language can be generated automatically, such as Java, C#, JavaScript or Python, with more to come. This code can then be integrated into any application (services, frontends, middleware).
```
Comment (NLueg) [inline README.md]:
```suggestion
The rules recorded in natural language are readable not only by humans but also by machines and therefore no longer need to be programmed by a software developer. This task is now taken over by openVALIDATION compiler. With integrated code generators, corresponding source code in the desired programming language can be generated automatically, such as Java, C#, JavaScript or Python, with more to come. This code can then be integrated into any application (services, frontends, middleware). 
```

File: README.md (hunk 1 above)
Comment (NLueg) [inline README.md]:
```suggestion
openVALIDATION enables the programming of validation rules using natural language, such as German or English. 
```

File: README.md [hunk 3]
```diff
@@ -1,140 +1,375 @@
...
     this.validate = function(model){
         return huml.validate(model);
     }
 }
+
 ```
-As this is a validation rule, we will throw an error *"your age HAS to be greater than 22"* if the criteria specified in the rule are not met.
 
-Besides raw strings the `--rule` flag also accepts paths to files containing the rules.
+The generated code does not depend on the 3'rd party libraries at all. Therefore, a custom framework is generated in addition to the rules. This framework contains a basic architecture to integrate the generated validation rules more easily into other systems.
```
Comment (NLueg) [inline README.md]:
```suggestion
The generated code does not depend on 3'rd party libraries at all. Therefore, a custom framework is generated in addition to the rules. This framework contains a basic architecture to integrate the generated validation rules more easily into other systems.
```

File: README.md [hunk 4]
```diff
@@ -1,140 +1,375 @@
...
+    <td>filtering the list</td>    
+  </tr>  
+  
+</table>
+
+<br/><br/><br/>
+## understand
+<br/>
+
+openVALIDATION enables programming of validation rules using natural language, such as German or English and many more.
```
Comment (NLueg) [inline README.md]:
```suggestion
openVALIDATION enables programming of validation rules using natural languages, such as German or English and many more.
```

File: README.md [hunk 5]
```diff
@@ -1,140 +1,375 @@
...
 
-Please refer to our [contribution guidelines](CONTRIBUTING.md).
+We still have 2 zeros in the version 0.0.X of openVALIDATION, so there is still a lot to do. If you want to join us, you are more than welcome to participate! 
 
-## Contact
+Check our [contribution guide](https://docs.openvalidation.io/contribution/contribution-guide).
+
+Even if you're not a developer or don't fully understand the technical part of openVALIDATION yet, it doesn't matter. There are many different ways to join the project. 
+
+Check our [contribution guide specially for beginners](https://docs.openvalidation.io/contribution/for-beginners)
```
Comment (NLueg) [inline README.md]:
```suggestion
Check our [contribution guide especially for beginners](https://docs.openvalidation.io/contribution/for-beginners)
```

File: README.md [hunk 6]
```diff
@@ -1,140 +1,375 @@
...
+
+<!-- generate new contributor list.. https://contributors-img.firebaseapp.com/ -->
+<a href="https://github.com/openvalidation/openvalidation/graphs/contributors"><img src="https://contributors-img.firebaseapp.com/image?repo=openvalidation/openvalidation"/>
+</a>
+
+<br/><br/><br/>
+## contact
+<br/>
 
 You can write an [E-Mail](mailto:validaria@openvalidation.io), mention our twitter account [@openVALIDATION](https://twitter.com/openVALIDATION) or message us at our instagram account [@openvalidation_](https://www.instagram.com/openvalidation_/).
```
Comment (NLueg) [inline README.md]:
```suggestion
You can write an [E-Mail](mailto:validaria@openvalidation.io), mention our twitter account [@openVALIDATION](https://twitter.com/openVALIDATION) or message us at our Instagram account [@openvalidation_](https://www.instagram.com/openvalidation_/).
```

File: README.md [hunk 7]
```diff
@@ -1,140 +1,375 @@
...
+
+<br/><br/><br/>
+## license
+<br/>
+
+openVALIDATION was initially developed as part of a research project at [BROCKHAUS AG](http://brockhaus-ag.de) in Dortmund.
+
+Only an Open Source solution has the ability to unfold its true potential. That's why we released it on GitHub as an open source project under the Apache 2.0 license.
```
Comment (NLueg) [inline README.md]:
```suggestion
Only an Open Source solution can unfold its true potential. That's why we released it on GitHub as an open-source project under the Apache 2.0 license.
```

**Comment by thecodemonkey** [comment]
@NLueg thanks!

**Pull request merged**